packages/rich-text-types/src/nodeTypes.ts

```diff
@@ -207,11 +207,16 @@ export interface TableCell extends Block {
   /**
    * @minItems 1
    */
-  content: Paragraph[];
+  content: Paragraph[] | OrderedList[] | UnorderedList[];
```

Comment:
This makes a huge difference in the schema. It should instead be `(Paragraph | OrderedList | UnorderedList)[]`